git-cheatsheet/lang/en.json

```diff
@@ -106,6 +106,10 @@
             "cmd": "branch --track <new branch name> <remote>/<branch>",
             "docs": "Create a new local branch from a remote-tracking branch."
         },
+        "init x": {
```

Comment:
I agree that this is a common command. I tend to use it from canned scripts (like Github that tells me how to run it), so I haven't thought of adding it, but I'm happy to.